packages/graphql-language-service-server/src/GraphQLCache.js

```diff
@@ -155,20 +149,18 @@ export class GraphQLCache {
   };
 
   getFragmentDefinitions = async (
-    graphQLConfig: GraphQLConfigInterface,
-    appName: ?string,
+    projectConfig: GraphQLProjectConfig,
   ): Promise<Map<string, FragmentInfo>> => {
     // This function may be called from other classes.
     // If then, check the cache first.
-    const rootDir = graphQLConfig.getRootDir();
+    const rootDir = projectConfig.configDir;
```

Comment:
I think you forgot to do a getter for `configDir` in https://github.com/graphcool/graphql-config/blob/master/src/GraphQLProjectConfig.ts, which makes me feel a little uncomfortable. If possible could we be consistent and use the proper getter? Not really a blocker though